### ISSUE TITLE
figDesc and alt text

### ISSUE THREAD

**Issue by aewingate** [IUBLibTech/newton_chymistry#105]
Another rendering issue we discovered in the review work is that the contents of <figDesc> are being displayed on Carbon instead of being hidden as alt text as in the current live site. The example below is from ALCH00051 (Keynes 62). 

Live site  https://webapp1.dlib.indiana.edu/newton/mss/norm/ALCH00051/ 
![image](https://user-images.githubusercontent.com/72820848/113041271-c5230e00-9167-11eb-8a19-d80989a8129a.png)

Carbon  http://carbon.dlib.indiana.edu:8220/text/ALCH00051/diplomatic
![image](https://user-images.githubusercontent.com/72820848/113041746-50040880-9168-11eb-8338-b3e7287b9ea8.png)

The note in angle brackets about the illustrations should be there, but the viewer shouldn't be able to see the text in the red boxes unless the image doesn't load or they're using a screen reader.

**Comment by Conal-Tuohy** [comment]
Currently the HTML presentation converts a TEI `figure` to an HTML `figure`, a TEI `figDesc` to an HTML `figCaption`, and a TEI `graphic` to an HTML `img`.
https://github.com/IUBLibTech/newton_chymistry/blob/master/xslt/p5-to-html.xsl#L424-L441
If you prefer not to use HTML `figCaption` for the `figDesc`, and instead to render the TEI `figDesc` only as an HTML `graphic/@alt`, you can replace the `figDesc` template with one that does nothing, e.g. `<xsl:template match="figDesc"/>`, and modify the `graphic` template so that it adds the `@alt` attribute as well as the `@src` attribute it already does. e.g. insert `<xsl:attribute name="alt" select="parent::figure/figDesc"/>`

**Comment by tubesoft** [comment]
I coded just as @Conal-Tuohy commented. It seems to work pretty much! Thank you, Con!

<img width="1179" alt="image" src="https://user-images.githubusercontent.com/17519127/113894707-4432ca80-9796-11eb-957a-92bcff623a4c.png">
<img width="663" alt="image" src="https://user-images.githubusercontent.com/17519127/113894783-54e34080-9796-11eb-9136-80d4e49b082a.png">

**Comment by wehooper** [comment]
The alt text values are no longer appearing so the issue has been resolved.
